projects/packages/videopress/src/client/block-editor/blocks/video/components/poster-panel/index.tsx

```diff
@@ -347,9 +422,22 @@ export default function PosterPanel( {
 	isGeneratingPoster,
 }: PosterPanelProps ): React.ReactElement {
 	const { poster, posterData } = attributes;
+
 	const [ pickFromFrame, setPickFromFrame ] = useState(
 		attributes?.posterData?.type === 'video-frame'
 	);
+	const [ previewOnHover, setPreviewOnHover ] = useState(
+		attributes?.posterData?.previewOnHover || false
+	);
+	const [ previewAtTime, setPreviewAtTime ] = useState(
+		attributes?.posterData?.previewAtTime || posterData?.atTime || 0
+	);
+	const [ previewLoopDuration, setPreviewLoopDuration ] = useState(
+		attributes?.posterData?.previewLoopDuration || DEFAULT_LOOP_DURATION
+	);
```

Comment:
I think we can polish and tidy this part. Something that I started in L426. Let me improve in a follow-up.